test/parallel/test-http-server-keep-alive-timeout-slow-headers.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
FWIW this filename is already exceptionally long :)
Though `...-client-headers` might convey more information about what the test does, yes.